.pre-commit-config.yaml

```diff
@@ -1,28 +1,28 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.2.3
+    rev: v2.4.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
+    -   id: check-docstring-first
     -   id: check-yaml
     -   id: debug-statements
+    -   id: check-ast
 -   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.7.1
+    rev: 3.7.9
     hooks:
     -   id: flake8
-        args: [
-          '--max-line-length=119',
-          '--ignore=E203,E266,E501,W503',
-          '--max-complexity=18',
-          '--select=B,C,E,F,W,T4,B9'
-        ]
-        exclude: __pycache__
--   repo: https://github.com/psf/black
-    rev: 19.3b0
+        args:
+          - --max-line-length=500
+          - --ignore=E203,E266,E501,W503
+          - --max-complexity=18
+          - --select=B,C,E,F,W,T4,B9
+-   repo: https://github.com/ambv/black
```

Comment:
Probably, better change order: firstly make format by black and only after running flake8, because if flake8 will find errors and after black will format files, then flake8 error message will be with wrong line position (after black, error will be on another position)

Reply:
Good suggestion, I fixed it! 👍